concent_api/common/tests/test_api_view.py

```diff
@@ -222,6 +222,8 @@ def setUp(self):
         compute_task_def['deadline'] = message_timestamp
         compute_task_def['extra_data'] = {
             'frames': [1],
+            'output_format': "PNG",
```

Comment:
Why mixing two different types of quotes ? Please choose one and keep it consistent. I think we use more of `''` in project than `""`.

Reply:
done